vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ExperimentalDesignDetailComponent.java

```diff
@@ -32,27 +33,39 @@ public class ExperimentalDesignDetailComponent extends Composite<CardLayout> {
   private static final long serialVersionUID = -2255999216830849632L;
   private static final String TITLE = "Experimental Design";
   private final Button createDesignButton = new Button("Add");
-  private final ExperimentCreationDialog experimentCreationDialog = new ExperimentCreationDialog();
+  private final ExperimentCreationDialog experimentCreationDialog;
   private final transient Handler handler;
-  private final VerticalLayout contentLayout = new VerticalLayout();
   private final VirtualList<Experiment> experiments = new VirtualList<>();
   private final CardLayout experimentalDesignAddCard = new ExperimentalDesignAddCard();
   private final ComponentRenderer<Component, Experiment> experimentCardRenderer = new ComponentRenderer<>(
-      ExperimentalDesignCard::new);
+      experiment -> {
+        ExperimentalDesignCard card = new ExperimentalDesignCard(experiment);
+        card.removeClassName("shadow-l");
+        card.addClassName("shadow-s");
+        return card;
+      });
```

Comment:
Nice temporary solution. 
As discussed we'll need to adjust how the cardlayout is styled to avoid statements such as this. 😓

Reply:
sadly this did not work as intended 😇 
```suggestion
      experiment -> {
        return new ExperimentalDesignCard(experiment);
      });
```